src/app/app.component.ts

```diff
@@ -1,11 +1,101 @@
 import { Component } from '@angular/core';
+import { Observable, timer, Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
+  seconds = 0;
+  minutes = 0;
+  hours = 0;
+  buttonStatus = 'Start';
+  isRunning = false;
+  waitStatus = false;
+  showSeconds = '00';
+  showMinutes = '00';
+  showHours = '00';
+  obsTimer$: Observable<number> = timer(0, 1000);
+  subscription!: Subscription;
+  timeFirstClick!: number;
+  
+  
+  startStop(status:boolean){
+    console.log(status);
+    if(status && !this.waitStatus){
+      this.isRunning = status;
```

Comment:
as I understand we just toggling this value? then maybe we don't need func param, we can use
`this.isRunning != this.isRunning` ?

Reply:
?